src/Omega_h_int_scan.cpp

```diff
@@ -27,7 +27,8 @@ LOs offset_scan(Read<T> a, std::string const& name) {
   out.set(0, 0);
 #if defined(OMEGA_H_USE_KOKKOS)
   auto outSub = Kokkos::subview(out.view(),std::make_pair(1,out.size()));
-  assert(outSub.size()==a.size());
+  LO outSub_size = outSub.size(); 
+  assert(outSub_size==a.size());
```

Comment:
This will still generate a warning in an opt build, since `outSub_size` is unused. You may want to use static cast inside the assert instead:
```
assert(static_cast<LO>(outSub.size())==a.size());
```

Reply:
@bartgol : good call!  I just made the change.